### DISCUSSION TITLE
Gists on Github mobile

### DISCUSSION THREAD

**Discussion by tusharsnx** [community/community#18795] [Mobile]
I could not find an option to see gists created by me. Is it not available on mobile platform yet or am I missing something?

**Comment by thomscoder** [comment]
I don't think it is possible yet in Github app version 1.69.1 Android.

It is very well possible by reaching the [Gist page](https://gist.github.com) even on mobile browsers

**Reply by heatblazer** [reply]
Is gist feature available now ? If not is there any chance to happen soon?

**Reply by TPS** [reply]
@heatblazer https://github.com/community/community/discussions/3331#discussioncomment-350789 🙄 & https://github.com/community/community/discussions/18795#discussioncomment-10490233

**Comment by Crapshit** [comment]
I have also the problem with latest 1.155.1 GitHub app for android.
I would love to have access to gist in the mobile app.

**Reply by Brndngbzl** [reply]
2 years since the post has been created and you're still unable to view gists in the GitHub app.

**Comment by diraneyya** [comment]
I would like to upvote this topic. As a coding teacher who masters the art of teaching markdown to young people this feature, creating and viewing gists on mobile phones, can be a game changer. We can use it for teaching children in 3rd world countries, for giving feedback and so on. It can serve as the perfect entry for git education for young adults.

**Comment by Saravishuakarma** [comment]
Where is option in github in my phone

**Comment by simplyequipped** [comment]
Also requesting GitHub mobile app support for gists on Android.

**Comment by MoroAmr** [comment]
We need gists for android please

**Comment by adoyle-h** [comment]
We need the gists on Github Mobile

**Comment by mkdizajn** [comment]
for anyone there is a octodroid app - that has support for gists

**Reply by papapahadi** [reply]
is it not on playstore?

**Comment by darianmorat** [comment]
+1 for Gists on Github Mobile.

**Comment by voroninadm** [comment]
+1 for Gists on Github Mobile.

**Comment by AmirAli-BahramJerdi** [comment]
On the mobile platform, there isn’t a dedicated option to view Gists you’ve created, like there is on the desktop version. To access your Gists on mobile, you can manually navigate to your GitHub profile in a web browser, and then add `/gists` to the URL (e.g., `github.com/yourusername/gists`). This will display your Gists. Alternatively, you might consider using the desktop site option in your mobile browser to access the full features of GitHub.

**Reply by Crapshit** [reply]
I would like to have that feature directly in the app, to avoid using a browser with login process and that.
For this I have the app 😊

**Reply by voroninadm** [reply]
How strange that gists not available on mobile. The mobile app is  abandoned?

**Reply by TPS** [reply]
> How strange that gists now available on mobile. The mobile app is  abandoned? 

The app gets updated w/ features requested by corporate users, & few else.

**Reply by diraneyya** [reply]
This is unfortunately true. Education and amateur users are likely to want to make commits and browse code from the app, while corporate users are likely to want to use the issue and other code management features from mobile

**Comment by hardik1408** [comment]
+1 for gists on mobile app

**Comment by Sachin2815** [comment]
I don't think it is possible yet in Github app version 1.69.1 Android.

**Comment by Sachin2815** [comment]
You'll need to use the desktop version to access this feature.

**Reply by ghost** [reply]
> You'll need to use the desktop version to access this feature.

Yes, we know. That's why we're asking GitHub to implement it on the mobile app.

**Comment by TPS** [comment]
I just realized official response (from 4ya!) is https://github.com/community/community/discussions/3331#discussioncomment-350789

**Comment by brandon-fryslie** [comment]
Why is literally the one feature that makes any sense to use on mobile not available?  Who is actually filing GitHub issues from their phone?  I could see a repo maintainer MAYBE giving a short reply to something on mobile…any other use case feels wildly optimistic.  We know a mobile app is expected regardless of how generally useless it will be.  It’s cool. 

But there’s one feature that has a clear mobile use case: I have a block of text I want to share and want a quick way to share it.  And it’s not implemented?  I’ve had this silly app installed for 2-3 years now and I was excited!  Here’s my chance!  All the preparation is finally coming to fruition!  And I see that I navigate to GitHub in my browser and manually add /gists to the URL?  What the literal f?  

Not that I expect software to be well designed or useful, on the whole.  But I kinda at least expect it from GitHub.  I guess I’ll just spend an hour replacing the app with an iOS Shortcut.  Then I can even use it from the share sheet!

**Reply by diraneyya** [reply]
Nowadays I subscribe to this thread solely for entertainment. Thanks!

**Reply by rain-fade** [reply]
The lack of functionality in the mobile app is astounding tbh. Thanks for the entertaining send up, regardless.
